lib/fog/bin/google.rb

```diff
@@ -4,15 +4,15 @@ class << self
     def class_for(key)
       case key
       when :compute
-        Fog::Compute::Google
```

Comment:
This is now causing an "uninitialized constant" to be thrown if someone tries to instantiate the old way.

If it's not too much trouble, can you add some shims, e.g.:

```
module Fog
  module Compute
    module Google
      def self.new(*args)
        # Issue a deprecation warning
        warn "[DEPRECATION] `Fog::Compute::Google.new` is deprecated. Please use `Fog::Google::Compute.new` instead."
        
        # Redirect the call to the new namespace
        Fog::Google::Compute.new(*args)
      end
    end
  end
end
```

Reply:
Done in c7a31601854d682860353265d7322f0944a1cf81.